source/common/grpc/buffered_message_ttl_manager.h

```diff
@@ -4,23 +4,25 @@
 
 #include "envoy/event/dispatcher.h"
 
-#include "source/common/grpc/buffered_async_client.h"
-
 namespace Envoy {
 namespace Grpc {
 
+using BufferedMessageExpirationCallback = std::function<void(uint64_t)>;
+
 // This class is used to manage the lifetime of messages stored in BufferedAsyncClient. Messages
 // whose survival period has expired will be deleted from the Buffer.
```

Comment:
Could you add a bit to this description that talks about how this acts on a set of ids that are all expired at the same time? I think that might not be immediately obvious

Thanks!

Reply:
added